demosplan/DemosPlanCoreBundle/Entity/User/User.php

```diff
@@ -540,7 +540,7 @@ public function getUsername()
     /**
      * Symfony > 6 needs getUserIdentifier() for auth system.
      */
-    public function getUserIdentifier(): ?string
+    public function getUserIdentifier(): string
     {
         return $this->getLogin();
```

Comment:
I see your point, symfony 6 enforces this method to return a string. So I'd rather return an empty string instead of null when nothing is set  4dbb3448 wdyt?